README.md

```diff
@@ -0,0 +1,177 @@
+- **Add a book:** You can add a new book to the store by entering its title and author information in the provided fields and submitting the form.
+- **Responsive design:** The app is designed to work seamlessly across desktop and mobile devices.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>  -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+Before you begin, make sure you have the following prerequisites:
+- A modern web browser such as Google Chrome or Mozilla Firefox.
+- An internet connection.
+- The latest version of Book Town project code.
+
+### Setup
+
+Clone this repository to your desired folder:
+-  `git clone git@github.com:christianonoh/book-town.git`
+-  `cd book-town`
+
+### Install
+#### Install the project dependencies by running the command:
+- `npm install`
+#### To run the project, execute the following commands:
+- `npm run build`
+- `npm start`
+- The Book Town application will now be running at `http://localhost:3000`
+
+### Usage
+#### Display a List of Books
+To display a list of books, simply navigate to the homepage of the Book Town application. You should see a list of books displayed on the page, along with their titles and authors.
+
+#### Add a Book
+To add a new book to the Book Town application, follow these steps:
+- Click on the "Add Book" button located at the top of the page.
+- Enter the title and author of the book in the fields provided.
+- Click on the "Add Book" button to submit the book.
+
+The new book should now be added to the list of books on the homepage.
+
+#### Remove a Selected Book
+To remove a selected book from the Book Town application, Click on the "Remove" button located next to the book you wish to remove.
+
+The selected book should now be removed from the list of books on the homepage.
+
+### Run tests
+
+- No tests available yet.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Christian Onoh**
+
+- GitHub: [@christianonoh](https://github.com/christianonoh)
+- Twitter: [@onohchristian](https://twitter.com/onohchristian)
+- LinkedIn: [Christian Onoh](https://www.linkedin.com/in/christianonoh)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add comments**
+- [ ] **Track progress**
+- [ ] **Responsive design** 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/christianonoh/math-wizardry/issues).
```

Comment:
- [ ] You did a great job working on your project documentation! 💯 However, please make sure that this link matches the correct project! 🤓 🔧